.checkpatch.conf

```diff
@@ -8,5 +8,7 @@
 --ignore=BIT_MACRO
 --ignore=PREFER_PRINTF
 --ignore=PREFER_SCANF
+--ignore=VOLATILE
```

Comment:
I was in a similar situation. Maxim and I tried to figure out what happened. I don't think we understood. Ultimately, we did interactive rebase (git rebase -i HEAD~x) and dropped the unrelated commits and force pushed the changes.